pootle/runner.py

```diff
@@ -148,16 +185,14 @@ def run_app(project, default_settings_path, settings_template,
                                           "'postgresql'" % args.db)
 
         try:
-            init_settings(config_path, settings_template, args.db)
+            init_settings(config_path, settings_template,
+                          args.db, args.db_name, args.db_user,
+                          args.db_password, args.db_host, args.db_port)
         except (IOError, OSError) as e:
             raise e.__class__('Unable to write default settings file to %r'
                 % config_path)
 
-        if args.db in ["mysql", "postgresql"]:
-            print("Configuration file created at %r: you must now update "
-                  "the settings for %s database" % (config_path, args.db))
```

Comment:
i did consider this but that would actually be a valid configuration.
Ill add a warning along the lines of - "you have not specified a password for your database. you may want to update the settings now"